### PR TITLE
Include AWS S3 for dynamic settings

### PR DESCRIPTION
**Pull request by rafaelgotts**

### CODE REVIEW

**Comment by rafaelgotts** [comment]
image

**Comment by drgarcia1986** [comment]
AWESOME, thank a lot, @rafaelgotts !
Could you fix this strange behavior of tests on Travis? (all s3 tests was skipped)

**Comment by rafaelgotts** [comment]
@drgarcia1986 Looks like is a problem with Travis Ubuntu images, see this [issue](https://github.com/travis-ci/travis-ci/issues/7940)

Now the coverage still on 100%.

I see that codacy complains about a new method too, and i wanna understand why they do this. Can you help me?

File: README.rst [hunk 1]
```diff
@@ -431,7 +431,7 @@ DATABASE
 
 You can read your settings dynamically form a database if you activate
 the ``DYNAMIC_SETTINGS`` special setting with the ``database`` backend
-(uses `sqlalchey <http://docs.sqlalchemy.org/>`__ library)
+(uses `sqlalchemy <http://docs.sqlalchemy.org/>`__ library)
```
Comment (drgarcia1986) [inline README.rst]: thank you

**Comment by drgarcia1986** [comment]
Thanks a lot @rafaelgotts ❤️

**Pull request merged**